index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio-setup-and-mobile-version-skeleton</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <header>
+    <div class="name">Carlos Herver</div>
+    <div class="button"><img src="assets/Union.png" alt="Hamburger button"> </div>
+  </header>
```

Comment:
Nice work implementing the logo and hamburger button, but I would like to suggest the following changes for the header section;
- [ ] Kindly put the name which is the **_logo_** in an `a` tag. The logo of any website should be clickable and most times direct users to the _**HomePage**_ of any website. It does not need to redirect to the HomePage now, but it would be nice to put it in the `a` tag.
- [ ] Kindly put the hamburger button in a `nav` tag. This would make the code semantic and easy to pick up by screen readers.
- [ ] I noticed the white patch at the right corner of the `Header`, it would be nice if you could fix that patch 😃 
![image](https://user-images.githubusercontent.com/77106962/143395009-0c95bc9c-d123-4615-8963-73a7b3b7ec17.png)

Reply:
Thanks! I believe the only question that I have is this: Can you solve the patch the way I did it? (setting width to 100% so any screen can fit)